src/lib/Components/Bidding/Components/Timer.tsx

```diff
@@ -18,10 +21,23 @@ export class Timer extends React.Component<TimerProps, TimerState> {
   constructor(props) {
     super(props)
 
-    const { liveStartsAt, endsAt } = props
-    const timeLeftInMilliseconds = Date.parse(liveStartsAt || endsAt) - Date.now()
+    const { liveStartsAt, endsAt, isPreview, startsAt } = props
 
-    this.state = { timeLeftInMilliseconds }
+    let relevantDate
+    let upcomingLabel
+
+    if (isPreview) {
```

Comment:
I ended up consolidating this logic here, since it affects both the `timeLeftInMilliseconds` and the `upcomingLabel`, but happy to move if this is unconventional.